### DISCUSSION TITLE
Welcome to immersions Discussions!

### DISCUSSION THREAD

**Discussion by bruchansky** [bruchansky/immersions#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RAMPAGEXP** [comment]
Hey Christophe, I'm an AR/VR game and experience developer, worked with a lot of Unity and Unreal engine projects, but as you said the internet is still the best metaverse out there, so I've been using 3js, babylon.js, AR.js and A-frame to build web experiences. Your project gave me some pretty good ideas for my structures. Hope to contribute as best as possible!

**Reply by bruchansky** [reply]
Hey @RAMPAGEXP thanks for the note, let me know if you have any questions or feedback.
